web/tests/sql.spec.ts

```diff
@@ -0,0 +1,467 @@
+  await workspace.clear();
+  await workspace.addBox({ id: 'eg0', name: 'Create example graph', x: 100, y: 100 });
+  await workspace.addBox({
+    id: 'sql', name: 'SQL1', x: 100, y: 200
+  });
+  await workspace.connectBoxes('eg0', 'graph', 'sql', 'input');
+  const se = await workspace.openBoxEditor('sql');
+  const tableBrowser = se.getTableBrowser();
+  await tableBrowser.toggle();
+  await tableBrowser.expectNode([0], 'edge_attributes', '`edge_attributes`');
+  await tableBrowser.expectNode([1], 'edges', '`edges`');
+  await tableBrowser.expectNode([2], 'graph_attributes', '`graph_attributes`');
+  await tableBrowser.expectNode([3], 'input.edge_attributes', '`input.edge_attributes`');
+  await tableBrowser.expectNode([4], 'input.edges', '`input.edges`');
+  await tableBrowser.expectNode([5], 'input.graph_attributes', '`input.graph_attributes`');
+  await tableBrowser.expectNode([6], 'input.vertices', '`input.vertices`');
+  await tableBrowser.expectNode([7], 'vertices', '`vertices`');
+  await tableBrowser.toggleNode([7]);
+  await tableBrowser.expectNode([7, 0], '*ALL*');
+  await tableBrowser.expectNode([7, 1], 'age (Double)', '`age`');
+
+  await tableBrowser.toggleFullyQualify();
+  await tableBrowser.expectNode([7, 1], 'age (Double)', '`vertices`.`age`');
+  await tableBrowser.expectNode(
+    [7, 0],
+    '*ALL*',
+    '`vertices`.`age`,\n' +
+    '`vertices`.`gender`,\n' +
+    '`vertices`.`id`,\n' +
+    '`vertices`.`income`,\n' +
+    '`vertices`.`location`,\n' +
+    '`vertices`.`name`');
+  await tableBrowser.toggleFullyQualify();
+});
+
+test('SQL2 box table browser', async () => {
+  await workspace.clear();
+  await workspace.addBox({ id: 'eg0', name: 'Create example graph', x: 100, y: 100 });
+  await workspace.addBox({ id: 'eg1', name: 'Create example graph', x: 350, y: 100 });
+  await workspace.addBox({
+    id: 'sql', name: 'SQL2', x: 100, y: 200
+  });
+  await workspace.connectBoxes('eg0', 'graph', 'sql', 'one');
+  await workspace.connectBoxes('eg1', 'graph', 'sql', 'two');
+  const se = await workspace.openBoxEditor('sql');
+  const tableBrowser = se.getTableBrowser();
+  await tableBrowser.toggle();
+  await tableBrowser.expectNode([0], 'one.edge_attributes', '`one.edge_attributes`');
+  await tableBrowser.expectNode([1], 'one.edges', '`one.edges`');
+  await tableBrowser.expectNode([2], 'one.graph_attributes', '`one.graph_attributes`');
+  await tableBrowser.expectNode([3], 'one.vertices', '`one.vertices`');
+  await tableBrowser.expectNode([4], 'two.edge_attributes', '`two.edge_attributes`');
+  await tableBrowser.expectNode([5], 'two.edges', '`two.edges`');
+  await tableBrowser.expectNode([6], 'two.graph_attributes', '`two.graph_attributes`');
+  await tableBrowser.expectNode([7], 'two.vertices', '`two.vertices`');
+  await tableBrowser.toggleNode([3]);
+  await tableBrowser.expectNode([3, 0], '*ALL*');
+  await tableBrowser.expectNode([3, 1], 'age (Double)', '`age`');
+});
+
+/*
```

Comment:
These are commented-out in the original. But let's fix what we can and delete what we no longer need during this migration.

Reply:
> These are commented-out in the original. But let's fix what we can and delete what we no longer need during this migration.

I've reviewed them. Nothing worth fixing I think. I've deleted them all.